packages/component-library/src/HeroImage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import { processStyle, createStyleBuilder, createBootstrap, StyleConfig as BaseStyleConfig } from './helpers';
+
+export interface Props {
+  id?: string;
+  className?: string;
+  children?: React.ReactNode;
+  style?: object;
+  url?: string;
+  [key: string]: any;
+}
+
```

Comment:
I think we also need a `StyleConfig` interface in here somewhere so the themed components can import it (maintaining the pattern that the other base components use). I think the same `StyleConfig` from `Button` would work

Reply:
Thanks, added with 9275fa2.